tf-modules/archive/bulk_operation.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_lambda_function" "bulk_operation" {
+  function_name    = "${var.prefix}-bulkOperation"
+  filename         = "${path.module}/../../packages/api/dist/bulkOperation/lambda.zip"
+  source_code_hash = filebase64sha256("${path.module}/../../packages/api/dist/bulkOperation/lambda.zip")
+  handler          = "index.handler"
+  role             = var.lambda_processing_role_arn
+  runtime          = "nodejs8.10"
+  timeout          = 300
+  memory_size      = 512
+  environment {
+    variables = {
+      METRICS_ES_HOST  = var.metrics_es_host
+      METRICS_ES_USER  = var.metrics_es_username
+      METRICS_ES_PASS  = var.metrics_es_password
+      GranulesTable    = var.dynamo_tables.granules.name
+      system_bucket    = var.system_bucket
```

Comment:
Maybe out of scope for this PR, but if you add `invoke = aws_lambda_function` to the list of environments here, similar to how it's done in the `api.tf` portion of the `archive` [module](https://github.com/nasa/cumulus/blob/master/tf-modules/archive/api.tf#L57), then the lambda function could be run stand-alone without having to be wrapped in a call to `async-operation`.

Reply:
```suggestion
      system_bucket    = var.system_bucket
      invoke                   = var.schedule_sf_function_arn
```